client-app/src/core/svc/NewsService.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import {XH, HoistService} from '@xh/hoist/core';
+
+/**
```

Comment:
Copy/Paste error in comment here .  Also, @HoistService() should be @HoistService